README.md

```diff
@@ -11,18 +11,18 @@ reporting.
 
 **This project is under active development.**
 
-VMS is live [here](http://52.8.110.63/).
+VMS is live [here](http://ec2-52-53-177-18.us-west-1.compute.amazonaws.com/en-us/).
 
-If you are an Outreachy Applicant, start with reading [this](https://github.com/systers/ossprojects/wiki/Volunteer-Management-System).
+If you are an Outreachy Applicant, start with reading [this](https://www.outreachy.org/communities/cfp/systers/).
 
 Features
 --------
 
-The VMS will be developed in phases. The initial implementation will track
+The VMS is developed in phases. The initial implementation will track
 volunteers' contact information, enable administrators to track volunteer
 hours, and allow reporting of useful information.
 
-The [requirements document](docs/Systers_GSoC14_VMS_Requirements.pdf) gives
+The [requirements document](https://github.com/anitab-org/vms/blob/develop/docs/GSoC18.md) gives
```

Comment:
This doc does not have info about the scope of the project. Removing this sentence would be better.

Reply:
Done, I removed